SIL.Windows.Forms/ClearShare/MetaData.cs

```diff
@@ -361,7 +362,7 @@ public bool FileFormatSupportsMetadata(string path)
 			return file != null && !file.GetType().FullName.Contains("NoMetadata");
 		}
 
-		public void Write(string path)
+		public void Write(string path, string originalImagePath = null)
```

Comment:
This seems... awkward. Seems like the metadata should instead pull in all it needs when first exposed to the original file. If for some reason that isn't possible, then it could at least keep the path it was given in FromFile around to use later if it's still available. That kind of state is undesirable, but also par for the course in the .net image api's.